ibis/expr/types/arrays.py

```diff
@@ -266,9 +266,16 @@ def repeat(self, n: int | ir.IntegerValue) -> ArrayValue:
 
     __mul__ = __rmul__ = repeat
 
-    def unnest(self) -> ir.Value:
+    def unnest(self,
+               offset: Optional[bool] = False
```

Comment:
```suggestion
               offset: bool = False
```

See my comment about `Optional` above.